src/module/vue/components/sf-moverow.vue

```diff
@@ -1,11 +1,12 @@
 <template>
   <div class="movesheet-row" :class="{ highlighted: move.highlighted }">
     <h4 class="flexrow" :title="tooltip">
-      <i
-        class="isicon-d10-tilt juicy clickable text nogrow"
-        style="padding-right: 0.5em"
-        @click="rollMove"
-      ></i>
+      <btn-rollmove
+        :disabled="!canRoll"
+        class="juicy text nogrow"
+        :actor="actor"
+        :move="move"
+      />
```

Comment:
```suggestion
      <btn-rollmove
        :disabled="!canRoll"
        class="juicy text nogrow"
        style="margin-right: 0.25em"
        :actor="actor"
        :move="move"
      />
```

Reply:
i'm a big fan of `gap`, personally - makes it easier to keep spacing/rhythm consistent, it applies to the parent so you can more easily control the layout's broad strokes, and it can still be adjusted manually with margin/padding when necessary. ive added a couple classes here so that it's a little more clear what CSS rules apply to what element (there's more work to be done there, i think, but i'll come back to this whole layout once ive done PRs for the other components of this layout)